test/RunTests

```diff
@@ -2473,4 +2473,8 @@ train-sets/ref/slates_simple_w_interactions.stderr
 {VW} --cb_adf --cache_file models/cb_test_cache.ldf.cache -d train-sets/cb_test.ldf
     train-sets/ref/cb_adf_mtr_cache_2.stderr
 
+# Test 344: check cb_adf with multiple passes
+{VW} --cb_adf -c --passes 2 -d train-sets/cb_adf_multipass_crash_1.data --holdout_off
```

Comment:
can you add a --cb test with --passes 2 too?

Reply:
--cb will reroute to cb_adf

Reply:
I do not think this code change will be hit by it? Since it is a single-line top learner. It generates the multi-ex and drives the base learner internally:

* Create multi_ex via [cbify_adf_data::copy_example_to_adf()](https://github.com/VowpalWabbit/vowpal_wabbit/blob/f2bd8e5e7f015e9fb852afb38bb084aaa3f9ae5b/vowpalwabbit/cbify.cc#L143)
* Invoke base learner [directly](https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/vowpalwabbit/cb_to_cb_adf.cc#L80)

Happy to add the test anyways.

Reply:
that changed with cb_to_cb_adf, check reduction stack of this test as an example: https://github.com/VowpalWabbit/vowpal_wabbit/blob/473f06263b874af1a2ff6103daf97765e29dec53/test/RunTests#L2461